projects/1.19/assets/dimensional-pockets-ii/dimensionalpocketsii/lang/zh_cn.json

```
@@ -0,0 +1,1072 @@
+	
+	"dimensionalpocketsii.gui.upgrade_station": "升级站模块",
+	"dimensionalpocketsii.gui.upgrade_station.craft_button": "点击以制造",
+	
+	"dimensionalpocketsii.gui.generator": "发电机模块",
+	"dimensionalpocketsii.gui.generator.mode_info": "更改发电模式",
+	"dimensionalpocketsii.gui.generator.mode_value": "发电模式：",
+	
+	"dimensionalpocketsii.gui.anvil": "铁砧模块",
+	
+	"dimensionalpocketsii.gui.focus": "次元焦点",
+	"dimensionalpocketsii.gui.focus.jump_info": "更改跳过模式",
+	"dimensionalpocketsii.gui.focus.jump_value": "当前的跳过模式：",
+	"dimensionalpocketsii.gui.focus.shift_info": "更改变换模式",
+	"dimensionalpocketsii.gui.focus.shift_value": "当前的变换模式：",
+	
+	"dimensionalpocketsii.gui.elytraplate.visor.dimension": "次元：",
+	"dimensionalpocketsii.gui.elytraplate.visor.empty": "无已安装的模块！",
+	
+	"dimensionalpocketsii.gui.elytraplate.settings.title": "鞘翅胸甲",
+	"dimensionalpocketsii.gui.elytraplate.settings.fly_info": "启用或禁用鞘翅飞行",
+	"dimensionalpocketsii.gui.elytraplate.settings.fly_value": "当前的鞘翅飞行：",
+	"dimensionalpocketsii.gui.elytraplate.settings.tele_info": "更改传送至方块模式",
+	"dimensionalpocketsii.gui.elytraplate.settings.tele_value": "当前的传送至方块：",
+	"dimensionalpocketsii.gui.elytraplate.settings.visor_info": "启用或禁用鞘翅胸甲面罩",
+	"dimensionalpocketsii.gui.elytraplate.settings.visor_value": "当前的鞘翅胸甲面罩：",
+	"dimensionalpocktesii.gui.elytraplate.settings.visor_warn": "警告！这种覆盖物会导致性能损失！",
+	"dimensionalpocketsii.gui.elytraplate.settings.solar_info": "启用或禁用太阳能发电机",
+	"dimensionalpocketsii.gui.elytraplate.settings.solar_value": "当前的太阳能发电机：",
+	"dimensionalpocketsii.gui.elytraplate.settings.charger_info": "启用或禁用物品充电器.",
+	"dimensionalpocketsii.gui.elytraplate.settings.charger_value": "当前的物品充电器：",
+	
+	"dimensionalpocketsii.gui.elytraplate.screen.title": "鞘翅胸甲口袋连接",
+	
+	"dimensionalpocketsii.gui.elytraplate.ender_chest.title": "鞘翅胸甲末影箱链接",
+	
+	"dimensionalpocketsii.gui.button.tank_clear": "按住潜行键并右击以清空流体储罐",
+	"dimensionalpocketsii.gui.button.tank_clear_shift": "警告！此变更无法撤回！",
+	"dimensionalpocketsii.gui.fluid_bar.pre": "流体储量：",
+	"dimensionalpocketsii.gui.fluid_bar.suff": " mB.",
+	"dimensionalpocketsii.gui.energy_bar.pre": "能量储量：",
+	"dimensionalpocketsii.gui.energy_bar.suff": " FE.",
+	
+	"dimensionalpocketsii.gui.config.name": "次元口袋 II 配置",
+	"dimensionalpocketsii.gui.config.general_title": "综合",
+	"dimensionalpocketsii.gui.config.messages_title": "控制台消息",
+	"dimensionalpocketsii.gui.config.visual_title": "视觉",
+	"dimensionalpocketsii.gui.config.blocked_title": "限制的元素",
+	"dimensionalpocketsii.gui.config.missing": "缺失的描述",
+	
+	"dimensionalpocketsii.gui.config.height": "口袋内部高度",
+	"dimensionalpocketsii.gui.config.height_slide": "口袋内部高度：",
+	"dimensionalpocketsii.gui.config.height_info": "允许你能更改口袋内部的高度",
+	"dimensionalpocketsii.gui.config.height_info_two": "数字范围限制在 15 到 255 之间",
+	
+	"dimensionalpocketsii.gui.config.height_enhanced": "增强型口袋内部高度",
+	"dimensionalpocketsii.gui.config.height_enhanced_slide": "增强型口袋内部高度：",
+	"dimensionalpocketsii.gui.config.height_enhanced_info": "允许你能更改增强型口袋内部的高度",
```

```suggestion
	"dimensionalpocketsii.gui.config.height_enhanced_info": "允许你更改增强型口袋内部的高度",
```
原句`允许……能……`莫名不顺
所有相似句式一并更改

---

抱歉我语文不好（恼）